config.json

```diff
@@ -13,6 +13,10 @@
     "remote_hls_gstreamer": "souphttpsrc location=http://YOUR_HLSSTREAM_URL_HERE.m3u8 ! hlsdemux ! decodebin ! videoconvert ! videoscale ! appsink",
     "simulation": "--yolo_json public/static/placeholder/alexeydetections30FPS.json --video_file_or_folder public/static/placeholder/frames --isLive true --jsonFps 20 --mjpgFps 0.2"
   },
+  "MONITOR_SETTINGS": {
+    "retry_delay_ms":30,
+    "max_retries_seconds":180
+  },
```

Comment:
"Monitor" seems very generic.

Also these settings as well as e.g. the JSON Ports are very specific to Darknet in my opinion. I therefore wonder if it is best to group them in a Darknet category somehow. e.g.

```JSON
{
    "DARKNET_CONFIG": {
        "retry_delay_ms": 30,
        "max_retries_seconds": 180,
        "json_stream": "http://localhost:8070",
        "mjpeg_stream": "http://localhost:8090"
    }
}
```

I realize that this is part of a bigger refactoring. If you feel like this it to much effort I'm happy to take the PR without the changes in config as I believe not many people will change it anyway.